### ISSUE TITLE
Non-Admin Users cannot delete items in the duplicate finder

### ISSUE THREAD

**Issue by Dpavlakis** [immich-app/immich#10162]
### The bug

When trying to use the duplicate finder as a non-admin user the following error is shown in the log 

[Nest] 1106  - 06/11/2024, 1:40:08 PM    WARN [Api:AuthGuard~bxxn82an] Denied access to admin only route: /api/system-config

### The OS that Immich Server is running on

Linux 6ef00b1b058c 6.1.79-Unraid #1 SMP PREEMPT_DYNAMIC Fri Mar 29 13:34:03 PDT 2024 x86_64 x86_64 x86_64 GNU/Linux

### Version of Immich Server

1.106.1

### Version of Immich Mobile App

v1.106.1

### Platform with the issue

- [X] Server
- [X] Web
- [ ] Mobile

### Your docker-compose.yml content

```YAML
Official Unraid Immich App
```

### Your .env content

```Shell
Not Sure where to find this
```

### Reproduction steps

```bash
1. Login as non admin user
2. Utilitize
3. Duplicates
4. Try delete duplicate
...
```

### Relevant log output

```shell
[Nest] 1106  - 06/11/2024, 1:40:08 PM    WARN [Api:AuthGuard~bxxn82an] Denied access to admin only route: /api/system-config
```

**Comment by bo0tzz** [comment]
#10113

**Comment by alextran1502** [comment]
Fixed in 1.106.2
